lukemckinstry/ebus-breakable-toy-serverless
# add express and local dynamodb services

Adds a node/expressjs service to point at DynamoDB datastore. Also adds a docker compose service to run a local test instance of DynamoDB. 

## Test instructions

- Pre-req: Install dynamodb-admin in your local environment with `npm install -g dynamodb-admin`

 - Run `./scripts/server`

 - You should have admin access to the local DynamoDB services by running `DYNAMO_ENDPOINT=http://localhost:8000 dynamodb-admin`

 - Add data to the DynamoDB table either via the dynamodb-admin GUI or via POST to `http://localhost:4000/api/transit/` with payload in the following format:
```
{"id": "test123", "route_id": test123" }
```
 - Try a GET request  to `http://localhost:4000/api/transit/`  and the data added to the table should be visible
